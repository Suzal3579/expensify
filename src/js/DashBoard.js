import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const DashBoard = () => (
    <div>
        <h1>DashBoard</h1>
        <ExpenseListFilters />
        <ExpenseList />
        <ExpensesSummary />
    </div>
);

export default DashBoard;
