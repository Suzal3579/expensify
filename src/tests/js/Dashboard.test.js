import React from "react";
import toJSON from "enzyme-to-json";
import { shallow } from "enzyme";

import DashBoard from "../../js/DashBoard";

test("should reander Dashboard page", () => {
    const wrapper = shallow(<DashBoard />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});