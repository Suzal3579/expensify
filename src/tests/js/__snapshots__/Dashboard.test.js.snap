// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should reander Dashboard page 1`] = `
<div>
  <h1>
    DashBoard
  </h1>
  <Connect(ExpenseListFilters) />
  <Connect(Component) />
  <Connect(Component) />
</div>
`;
